src/Umbraco.SampleSite.Web/Views/Partials/blocklist/Components/Feature.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Core.Models.Blocks.BlockListItem>;
+@using ContentModels = Umbraco.Cms.Web.Common.PublishedModels;
+@{
+    var feature = (Feature)Model.Content;
+}
```

Comment:
```suggestion
@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Core.Models.Blocks.BlockListItem<Feature>>;
@using ContentModels = Umbraco.Cms.Web.Common.PublishedModels;
@{
    var feature = Model.Content;
}
```

Minor thing, but might be cool to show off in the demo site. 

If you use the generic type if `BlockListItem<T>` you don't have to cast the content later 😄

Reply:
Nice, will fix up 💪